RestApi/Sepes.RestApi/Services/StudyService.cs

```diff
@@ -64,6 +66,42 @@ public async Task<Study> Save(Study newStudy, Study based)
             return study;
         }
 
+        public async Task<Study> DeletePod(Pod pod)
```

Comment:
Please use the UpdatePods method to figure out what pod to delete. You can mostly use the same way it figures out what pods are new to find the one to delete.

Reply:
Maybe change UpdatePods to ManagePods and have it call CreatePod and DeletePod methods.